### PR TITLE
Refactor the boarding events for TRoW S8 Clearwater Port

### LINKED ISSUE #6298: TRoW S8 inconsistency when trying to board a ship as a soldier
### Game and System Information
 - 1.16.x (the rest doesn't matter)

### Describe the bug
There are three places where the ships can arrive at. 
(29,16) = a, (32,14) = b, and (31,16) = c.
For a and b, the 'narrator' admonishes the unit. At c it's Aethyr instead.
And I don't how that makes sense. It should be either Aethyr only, or narrator only.

### To Reproduce

1. Load the provided save.
2. Move units to the provided coordinates.
3. Note that narrator and Aethyr say the same, but the one talking depends on the coordinates.

### Expected behavior
Either Aethyr only, or narrator only.

### Savefiles
[TRoW-Clearwater Port-Auto-Save1.gz](https://github.com/wesnoth/wesnoth/files/7538782/TRoW-Clearwater.Port-Auto-Save1.gz)

### Additional context
c https://github.com/wesnoth/wesnoth/blob/60c6e72921fe91339014cd9cab49afde5affe2f7/data/campaigns/The_Rise_Of_Wesnoth/scenarios/08_Clearwater_Port.cfg#L778

b https://github.com/wesnoth/wesnoth/blob/60c6e72921fe91339014cd9cab49afde5affe2f7/data/campaigns/The_Rise_Of_Wesnoth/scenarios/08_Clearwater_Port.cfg#L832

a https://github.com/wesnoth/wesnoth/blob/60c6e72921fe91339014cd9cab49afde5affe2f7/data/campaigns/The_Rise_Of_Wesnoth/scenarios/08_Clearwater_Port.cfg#L887

### PR DESCRIPTION
**Pull request by Wedge009**
Resolves #6298

### CODE REVIEW

File: data/campaigns/The_Rise_Of_Wesnoth/scenarios/08_Clearwater_Port.cfg [hunk 1]
```diff
@@ -731,33 +731,35 @@
...
-                y=16
+                x={ABSCISSA}
+                y={ORDINATE}
             [/have_unit]
 
             [then]
                 [if]
                     [variable]
-                        name=First_Ship
+                        name={SHIP_EVENT}
```
Comment (stevecotton) [inline data/campaigns/The_Rise_Of_Wesnoth/scenarios/08_Clearwater_Port.cfg:755]: I think the `First_Ship`, `Second_Ship` and `Third_Ship` variables are old code that could be replaced by using `[remove_event]` when a ship departs.

Comment (Wedge009) [reply data/campaigns/The_Rise_Of_Wesnoth/scenarios/08_Clearwater_Port.cfg:755]: I don't understand what that means. All I'm trying to do here is remove the duplicate code, not change existing behaviour, although if there are deprecated patterns that we should be avoiding I can try to fix that, although remember I'm still a WML novice.

Comment (stevecotton) [reply data/campaigns/The_Rise_Of_Wesnoth/scenarios/08_Clearwater_Port.cfg:755]: Ah, sorry I assumed that you weren't a WML novice. I think I'd need to try writing the whole thing myself in order to explain what I mean.

Comment (Wedge009) [reply data/campaigns/The_Rise_Of_Wesnoth/scenarios/08_Clearwater_Port.cfg:755]: If you can improve the event itself, go ahead and add your changes on top of this. The intent was simply to satisfy #6298: three similar events that (apparently unintentionally) introduces an inconsistency between whether Commander Aethyr speaks, or the narrator. Removing the unnecessary duplication helps keeps the events consistent, that's all my intention was. Any further improvement on top of that is just a bonus and separate to this PR.

**Review by stevecotton — APPROVED** [review]

**Pull request merged**